homeassistant/components/google_generative_ai_conversation/conversation.py

```diff
@@ -152,6 +149,17 @@ def _escape_decode(value: Any) -> Any:
     return value
 
 
+def _chat_message_convert(
+    message: conversation.Content | conversation.NativeContent[genai_types.ContentDict],
+) -> genai_types.ContentDict:
+    """Convert any native chat message for this agent to the native format."""
+    if message.role == "native":
+        return message.content
+
+    role = "model" if message.role == "assistant" else message.role
```

Comment:
Why not just message.role in which case you can inline it below? See also other comment below.